lib/components/T3Form/T3FormFieldList/T3FormFieldList.vue

```diff
@@ -11,7 +11,7 @@
         :field="field"
       >
         <T3FormFieldList
-          v-if="field.fieldlist"
+          v-if="field.fieldlist && field.elements"
```

Comment:
`field.elements` is an array, so this condition will be truthy even when an array will be empty; I suggest adding `.length` like in the `restoreModel` method.

Reply:
in that case, API doesn't deliver an empty array, in the case of 0 elements API doesn't deliver elements field at all :D the empty array is not an issue, but the lack of this field is :D However, I can agree we don't have to render this component in case of the empty array - at least for now 👍